uiowa/uiowa
# Feature: Blockquote field updates with new basic text format

Resolves https://github.com/uiowa/uiowa/issues/5536. 

# How to test

`blt frontend`

- Sync a site with a lot of blockquotes `blt ds --site=tippie.uiowa.edu` check out https://tippie.uiowa.ddev.site/undergraduate/admissions for blockquotes. 
- Sync a site with some temp storage and verify that the changes have been made: `blt ds --site=sandbox.uiowa.edu` and go to https://sandbox.prod.drupal.uiowa.edu/node/656/layout and edit the first blockquote and verify that the word "President" has been removed from the Source field. 
   - ~One issue I've noticed is that the item in temp storage in this example has `<p>` tags around it which should not be allowed~ https://github.com/uiowa/uiowa/pull/5569/commits/c0d1eced8e298bdb6b55e263d4eb09d6dea20e32. 
- Verify that update hook runs and changes the Content and Source fields to text areas with `basic` html
- Review the new `basic` text format/permissions and let me know if I am missing any files or entries or splits. 
- Review the update hook.
